src/cpp/r/include/r/RInterface.hpp

```diff
@@ -51,6 +51,59 @@ typedef struct SEXPREC *SEXP;
 
 #endif
 
+typedef struct R_BCSTACK_T {
```

Comment:
Yes, we don't actually access this struct so technically we could emit the definition entirely and just replace with void pointers. But since we do include the struct definition it should be correct. I've updated it.